app/model/NotificationTestType.scala

```diff
@@ -17,11 +17,20 @@
 package model
 
 import model.ApplicationStatus.ApplicationStatus
-import model.ApplicationStatus.{ PHASE1_TESTS_FAILED, PHASE2_TESTS_FAILED, PHASE3_TESTS_FAILED}
+import model.ApplicationStatus.{ PHASE1_TESTS_FAILED, PHASE2_TESTS_FAILED, PHASE3_TESTS_FAILED, READY_FOR_EXPORT }
 import model.ProgressStatuses._
 
+sealed trait NotificationTestType {
```

Comment:
I think this trait name is a bit confusing - its a notification for a test, not a test notification.  Maybe NotificationForTest?  Not sure the `Type` on end adds anything either.